tests/endpoints/test_endpoints.py

```diff
@@ -0,0 +1,107 @@
+from app import db as _db
+from app.serotracker_sqlalchemy.models import DashboardSource
+
+
+def test_healthcheck(client):
+    response = client.get('/healthcheck/')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is str
+    assert data == 'The healthcheck endpoint was hit.'
+
+
+# Test get filter options
+def test_get_filter_options(client):
+    response = client.get('/data_provider/filter_options')
+    status_code = response.status_code
+    data = response.get_json()
+    assert status_code == 200
+    assert type(data) is dict
```

Comment:
any other assertions re the return that we should be making here? or does that assume that the form of filter_options doesn't change?

Reply:
thinking about this across the rest of the endpoints too - like, is there a way to make sure we're not just getting a janky empty dict back? or si that not the purpose of these things to test?

Reply:
we could store a list of keys that filter options should have and assert that `data` contains those.

Reply:
that's possible -- I then become worried about that being a fragile test if there is a change in our data. Not sure ow to straddle this line.

Reply:
Eventually filter_options endpoint will pull data from airtable - I think once that happens it should be easier to design a more robust test. 

I'm more inclined to leave this as is for now so that we don't spend time overengineering a test